packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -81,7 +81,7 @@ function runConflictFarmTests(opts: IConflictFarmConfig, extraSeed?: number): vo
             }
             mt.seedWithArray(seedArray);
 
-            const clients: TestClient[] = [new TestClient()];
+            const clients: TestClient[] = [new TestClient({ mergeTreeUseNewLengthCalculations: true })];
```

Comment:
Should we be concerned about losing coverage for the non-mergeTreeUseNewLengthCalculations case?

Reply:
We shouldn't mix it, however the results of mixing it won't be worse that than the results of having it off, as without it in the rare cases this fixes, we will already have decoherence. I've turned it on here just to keep coverage, it is off for the replay tests which cover very similar cases, and on in only a couple specific regression cases in applyMsg.spec.ts.